NEWS.md

```diff
@@ -119,7 +119,7 @@ This section lists changes that do not have deprecation warnings.
   * `transpose(::AbstractVector)` now always returns a `RowVector` view of the input (which is a
      special 1×n-sized `AbstractMatrix`), not a `Matrix`, etc. In particular, for
      `v::AbstractVector` we now have `(v.').' === v` and `v.' * v` is a scalar. ([#19670]).
-     To create a nx1-sized column vector out of a vector, use `reshape(v,length(v),1)`.
+     To create a 1-column vector out of a vector, use `reshape(v,length(v),1)`.
```

Comment:
the "nx1-sized" part was fine, it was saying "To create a ... vector out of a vector" that I was commenting on